include/zephyr/arch/arm/aarch32/cortex_m/scripts/linker.ld

```diff
@@ -135,6 +135,8 @@ SECTIONS
 	*(.text)
 	*(".text.*")
 	*(".TEXT.*")
+        *(.shared_irq)
+        *(.shared_irq_arg)
```

Comment:
this code together with:
https://github.com/zephyrproject-rtos/zephyr/blob/ffd7107810322e5de7f86b37ed88b2533cc0acdf/arch/common/shared_irq.ld#L11-L12

means you now get this twice in the linker script, in two output sections, like this:
```
 60     text :                                                                                                                                                                      
 61  {                                                                                                                                                                              
 62  __text_region_start = .;                                                                                                                                                       
 63  *(.text)                                                                                                                                                                       
 64  *(".text.*")                                                                                                                                                                   
 65  *(".TEXT.*")                                                                                                                                                                   
 66         *(.shared_irq)                                                                                                                                                          
 67         *(.shared_irq_arg)                                                                                                                                                      
 68  *(.gnu.linkonce.t.*)                 
....
180     rodata :                                                                                        
181  {                                                                                                  
182  *(.rodata)                                                                                         
183  *(".rodata.*")                                                                                     
184  *(.gnu.linkonce.r.*)                                                                               
185 . = ALIGN(4);                                                                                       
186 _shared_irq_start = .;                                                                              
187 KEEP(*(.shared_irq))                                                                                
188 KEEP(*(.shared_irq_arg))                                                                            
189 _shared_irq_end = .;         
...
```

that doesn't seem right.